components/supersaas/sources/changed-appointments.js

```diff
@@ -0,0 +1,39 @@
+const supersaas = require('../supersaas.app.js');
+
+module.exports = {
+  key: 'supersaas-changed-appointments',
+  name: 'New or changed appointments',
+  description: `This event source tracks new and changed appointment events from the selected schedules and emits them.`,
```

Comment:
In other components we normally use the pattern "Emits an event for every..." or "Emits an event for each new..." to make the language the user sees consistent in the UI. Could you try to re-word the description to fit that pattern?